### ISSUE TITLE
Fix complaint about Apple-specific ssh arguments

### ISSUE THREAD

**Issue by unixorn** [unixorn/zsh-quickstart-kit#193]
**Describe the bug**

On 12.4, we see a deprecation warning about ssh's Apple-specific flags.
```
WARNING: The -K and -A flags are deprecated and have been replaced
         by the --apple-use-keychain and --apple-load-keychain
         flags, respectively.  To suppress this warning, set the
         environment variable APPLE_SSH_ADD_BEHAVIOR as described in
         the ssh-add(1) manual page.
```

**Expected behavior**
No complaints about deprecation

**Desktop (please complete the following information):**
- OS/distro: macOS
- Version 12.4

**Comment by jfmcdowell** [comment]
Did PR #136 not fix this after all?

**Comment by unixorn** [comment]
Maybe the warning was added before Monterey and I only noticed it then?

**Comment by unixorn** [comment]
Closed by #228
